dagger/dag_creator/airflow/operators/spark_submit_operator.py

```diff
@@ -45,6 +43,10 @@ def emr_client(self):
     def ssm_client(self):
         return boto3.client("ssm")
 
+    @property
+    def emr_master_instance_id(self):
+        return
```

Comment:
Is this right? A property that returns nothing?

Reply:
My bad. Fixed in 
007ef64